gradle-plugin/plugin/build.gradle.kts

```diff
@@ -83,7 +83,6 @@ dependencies {
   implementation(libs.asm.commons)
 
   add(shadowedDependencies.name, libs.develocity.agent.adapters)
-  add(shadowedDependencies.name, libs.dexlib2)
   // Needed for the GradleRunner in the functional tests. We've had issues with the version of Guava
   // from one dependency conflicting with that of dexlib2, so we'll use the same version here.
```

Comment:
Not totally familiar with this issue but can follow up with a commit to remove or update the Guava dep cc @runningcode as I believe you added this comment

Reply:
I did not add this comment but happy to look in to it!

Reply:
Interesting, not a huge deal so we can kick it.